### PR TITLE
45 set header to register state and show appropriate header if user is logged in

### LINKED ISSUE #45: Set Header to Register State and Show Appropriate Header if User is Logged in
As a user I need to see appropriate buttons on the header whether I am logged in, logged out, or registering.  To do this, state of the header needs to be managed with using the 'loggedIn' variable and any other variables that should detect logged in state.

Acceptance Criteria:
* The header displays correctly whether a user is logged in or not
* The 'setLoggedIn' method is called upon a successful login to change the state to logged in

Hint: You may have to manage state within a higher level file that passes state variables or state management methods to lower level components

### PR DESCRIPTION
**Pull request by Magatsuken**
Header displays correctly when user is logged in and when user is not logged in

### CODE REVIEW

File: client/src/Header/Header.js [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
 import Navbar from 'react-bootstrap/Navbar';
-import {useState} from "react";
+import { useState } from "react";
 import Button from "react-bootstrap/Button";
+import {useNavigate} from "react-router-dom";
 
-export default function Header() {
+export default function Header({ isLoggedIn, setIsLoggedIn }) {
+    const onClick = () => {
+        setIsLoggedIn(isLoggedIn = false);
```
Comment (ndohertyjr) [inline client/src/Header/Header.js]: setIsLoggedIn parameter should just be false.  Like: setIsLoggedIn(false)

File: client/src/Login/Login.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ export default function Login() {
                 }
                 else {
                     console.log("Route to next page!");
+                    setIsLoggedIn(isLoggedIn = true);
```
Comment (ndohertyjr) [inline client/src/Login/Login.js]: setIsLoggedIn(true)

**Review by ndohertyjr — CHANGES_REQUESTED** [review]
Review setIsLoggedIn syntax

File: client/src/Header/Header.js (hunk 1 above)
Comment (Magatsuken) [reply client/src/Header/Header.js]: Done

File: client/src/Login/Login.js (hunk 2 above)
Comment (Magatsuken) [reply client/src/Login/Login.js]: Done

**Review by ndohertyjr — APPROVED** [review]

**Pull request merged**